From 8ae03093dc1a5f480ef3ad061d843232b6b18a1d Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 28 Sep 2018 14:43:20 +0800
Subject: [PATCH 4869/5242] MLK-19850-2 ARM64: dts: fsl-imx8dx: enlarge USB
 PHY register region

commit  bf477d1420c78a6e88334a3f30c5b44377d51a87 from
https://source.codeaurora.org/external/imx/linux-imx.git

There is a DCD module at USBPHY, the offset is from 0x800.

Reviewed-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index 3f2ba667..16428d9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -2366,7 +2366,7 @@
 
 	usbphy1: usbphy@0x5b100000 {
 		compatible = "fsl,imx8qm-usbphy", "fsl,imx7ulp-usbphy", "fsl,imx6ul-usbphy", "fsl,imx23-usbphy";
-		reg = <0x0 0x5b100000 0x0 0x200>;
+		reg = <0x0 0x5b100000 0x0 0x1000>;
 		clocks = <&clk IMX8QXP_USB2_PHY_IPG_CLK>;
 		power-domains = <&pd_conn_usbotg0_phy>;
 	};
-- 
1.7.9.5

